ext_emconf.php

```diff
@@ -26,12 +26,12 @@
             'crawler' => '9.0.0-9.9.99'
         ],
     ],
-    'state' => 'alpha',
+    'state' => 'stable',
     'clearCacheOnLoad' => 1,
     'author' => 'Philipp Kuhlmay',
```

Comment:
One more: I think you should add Tomas and me also to the authors ;-)

Reply:
I added you and Tomas